modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/BoundObjectDefinitionsExportImportTest.java

```diff
@@ -0,0 +1,260 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.web.internal.portlet.action.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.object.admin.rest.dto.v1_0.ObjectDefinition;
+import com.liferay.object.admin.rest.resource.v1_0.ObjectDefinitionResource;
+import com.liferay.portal.kernel.json.JSONArray;
+import com.liferay.portal.kernel.json.JSONFactory;
+import com.liferay.portal.kernel.json.JSONObject;
+import com.liferay.portal.kernel.json.JSONUtil;
+import com.liferay.portal.kernel.portlet.bridges.mvc.MVCActionCommand;
+import com.liferay.portal.kernel.portlet.bridges.mvc.MVCResourceCommand;
+import com.liferay.portal.kernel.test.util.TestPropsValues;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.portal.test.rule.FeatureFlags;
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.vulcan.pagination.Page;
+import com.liferay.portal.vulcan.pagination.Pagination;
+
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Guilherme Sá
+ */
+@FeatureFlags("LPS-187142")
+@RunWith(Arquillian.class)
+public class BoundObjectDefinitionsExportImportTest
+	extends BaseExportImportTestCase {
+
+	@ClassRule
+	@Rule
+	public static final LiferayIntegrationTestRule liferayIntegrationTestRule =
+		new LiferayIntegrationTestRule();
+
+	@Before
+	public void setUp() throws Exception {
+		user = TestPropsValues.getUser();
+
+		ObjectDefinitionResource.Builder builder =
+			_objectDefinitionResourceFactory.create();
+
+		_objectDefinitionResource = builder.user(
+			user
+		).build();
+
+		Class<?> clazz = getClazz();
+
+		_defaultObjectDefinitionJSONString = StringUtil.read(
+			clazz.getResourceAsStream(
+				"dependencies/test-default-object-definition-draft.json"));
+		_defaultObjectRelationshipJSONString = StringUtil.read(
+			clazz.getResourceAsStream(
+				"dependencies/test-default-object-relationship.json"));
+	}
+
+	@Test
+	public void testExportImportBoundObjectDefinitions() throws Exception {
+
+		// Import bound object definitions as draft
+
+		String draftObjectDefinitionsJSONString = String.valueOf(
```

Comment:
Can you rename it to draftObjectDefinitionsJSONArray?

Reply:
SF only allows using "JSONArray" for the actual `JSONArray` java type. I made this variable a String so that we don't have to call `toString` on it multiple times.